git_pile/config.py

```diff
@@ -77,6 +89,30 @@ def __init__(self):
             except:
                 warn(f"could not set {key}={value} from git config")
 
+    def normalize(self, root):
+        import sys
```

Comment:
Hm... Why is the import here instead of the usual place?

Reply:
leftover... there were no sys imported for my "XXXX" debugs :)